packages/server/src/sdk/app/rows/search/utils.ts

```diff
@@ -108,3 +108,18 @@ export function searchInputMapping(table: Table, options: RowSearchParams) {
   }
   return options
 }
+
+export function isSearchingByRowID(query: SearchFilters): boolean {
+  for (let searchField of Object.values(query)) {
+    if (typeof searchField !== "object") {
+      continue
+    }
+    const hasId = Object.keys(searchField).find(
+      key => key.endsWith("_id") && searchField[key]
```

Comment:
Why is this endsWith? Could this be conflicting with things such as `department_id` or any other column name ending with `_id`?

Reply:
True - I was trying to handle for prefixes - but I'll do it a different way!

Reply:
We have a util that trims the prefixes. We should use that